### DISCUSSION TITLE
Welcome to runit Discussions!

### DISCUSSION THREAD

**Discussion by fey** [hexlet-volunteers/runit#198] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by glazovanastasia** [comment]
Появился интересный вопрос/предложение. У меня не было учетной записи на сайте, но я через гитхаб одновременно могу её создать и авторизоваться. И вот захожу в свою учетку, хочу сменить пароль, но при регистрации я его не задавала, соответственно, смена невозможно, потому что для этого надо ввести текущий пароль. И всё, затык. Может, есть смысл убрать функцию смены пароля для пользователей, авторизованных ТОЛЬКО через гитхаб?

**Reply by glazovanastasia** [reply]
Ну либо я просто создаю ишью и открываю в аллюре дефект

**Reply by glazovanastasia** [reply]
Либо при авторизации через гитхаб параллельно на почту будет вот такое сообщение приходить
![image](https://github.com/hexlet-rus/runit/assets/127899807/a4254fa6-3322-4f28-9aba-b2f41a1e2d69)

**Reply by fey** [reply]
Обычно такое может решаться через "Забыл пароль", но это пока не доделали :(
Просто различать пользователей которые вошли через гитхаб или залогинились - немного неудобно. Иногда делают еще одну форму, когда осле регистрации указывают пароль, а бывает, как у нас, генерируется пароль.

**Reply by fey** [reply]
> Ну либо я просто создаю ишью и открываю в аллюре дефект

Можете создать issue с фича реквестом - добавить возможность привязывать гитхаб аккаунт к уже созданному профилю)
